CONTRIBUTING.md

```diff
@@ -207,19 +207,22 @@ dotnet build
 
 #### Run the Azure MCP server in HTTP mode
 
+> [!NOTE]
+> **Microsoft Employees Only:** This section is intended for Microsoft employees with access to the TME tenant. Any outside contributors or Microsoft employees without access to TME will need to modify their copy of `launchSettings.json`.
```

Comment:
Consider re-writing this to be the inverse, something like "Open source contributors need to do A,B,C.  If you are a Microsoft employee then please also review our .... for specific instructions.